src/js/components/ClassChoiceButton.js

```diff
@@ -1,16 +1,38 @@
 import React, { Component } from 'react';
 import { saveClass } from '../actions/nextStepCharAction';
+import { connect } from "react-redux"
+import "../../styles/attributes.scss";
 
-require("../../styles/attributes.scss");
-
+@connect((store) => {
+    return {
+        lastStep: store.nextStep
+    };
+})
 export default class ClassChoiceButton extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            cssClass: 'attribute-choice-button'
```

Comment:
You can define it as simple const in render function, not in state. It's better to have stateless components.